src/NUnitFramework/framework/Internal/Execution/IMethodValidator.cs

```diff
@@ -0,0 +1,16 @@
+using System.Reflection;
```

Comment:
Can you please add the copyright header to this file?

Reply:
Thanks, added